ci/build_container/Makefile

```diff
@@ -0,0 +1,44 @@
+# The individual build recipe scripts must contain sufficient information (e.g.  SHA, URL, repo,
+# version number, etc.) to uniquely identify the revision of the upstream dependency.  This allows
+# make to pick up changes with a simple direct dependency on the build recipe.
+# TODO(htuch): These dependencies should be installed into distinct --prefix directories, so we can
+# easily blow away the lib/ and include/ install directories when we upgrade version.
+
+all: $(THIRDPARTY_DEPS)/libevent.dep \
```

Comment:
nit: Can we fix the whitespace indent in this file. It's making my OCD flair up.